ci.sh

```diff
@@ -12,7 +12,8 @@ set -eu
 
 OS=`uname -s`
 
-MYDIR=$(dirname $(realpath "$0"))
+SELF=$(realpath "$0")
+MYDIR=$(dirname "${SELF}")
```

Comment:
(No action required)

While it can look a bit surprising at first, it’s technically possible to nest double quotes:

```suggestion
MYDIR=$(dirname "$(realpath "$0")")
```

Reply:
True. That works for `$()`, not sure about `${}`.

https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_03

So I decided to write the way that requires least amount of prior knowledge to read.